### DISCUSSION TITLE
Missing Tailwind Classes like: grid grid-cols-*

### DISCUSSION THREAD

**Discussion by sameraw** [tailpress/tailpress#227] [Q&A]
I am having an issue where i cannot use tailwind grid class and some other classes. 
**For example:**
`wp_enqueue_script( 'tailpress', tailpress_asset( 'js/app.js' ), array(), $theme->get( 'Version' ) );` 
**Causes the grid element to look like this:**
![Screenshot 2024-08-22 at 6 43 20 PM](https://github.com/user-attachments/assets/1cc7b579-3050-4ec9-93c4-28e74105b613)

When I remove it and add the regular tailwind library: 
`wp_enqueue_script( 'tailpress', tailpress_asset( 'js/tailwind.js' ), array(), $theme->get( 'Version' ) ); `
It looks fine:
![Screenshot 2024-08-22 at 6 43 29 PM](https://github.com/user-attachments/assets/92ab0303-2770-4a9c-b148-ba1971740745)
there seems to be missing CSS classes native to Tailwind.

**Comment by sameraw** [comment]
Try using the following:

```
<div class="grid grid-cols-2 md:grid-cols-5 gap-4">
<div>Some Text </div>
<div>Some Text </div>
<div>Some Text </div>
<div>Some Text </div>
<div>Some Text </div>
</div>
```

**Comment by jeffreyvr** [comment]
Hi, You have to specify which files Tailwind CSS needs to look at, you can define them here:

https://github.com/jeffreyvr/tailpress/blob/master/tailwind.config.js#L6

Of course, in addition to that you need to have the compiler running.

Using the JS version of Tailwind is not recommended for production, as noted here https://tailwindcss.com/docs/installation/play-cdn.
